tests/unit/testsuite/Bolt/Boltpay/Model/OrderTest.php

```diff
@@ -1693,6 +1693,19 @@ private function isBoltOrderSetUp($paymentMethod)
         return $orderMock;
     }
 
+    /**
+     * @test
+     * that when checked on an order without a paymenty, false is returned
+     *
+     * @covers ::isBoltOrder
```

Comment:
@johnmccombs1 Please of observe the changes that I made for future reference.  `@covers` is important to avoid false test positives.